### PR TITLE
updated foreign key fk_crops_id to fk_crop_id on crop_stages table an…

### PR DESCRIPTION
**Pull request by mahieakhtar**
…d seed file
Updated foreign key fk_crops_id to fk_crop_id on crop_stages table and seeding file.

### CODE REVIEW

**Comment by cecastosic** [comment]
This change will also effect several controllers, endpoints will not work.

File: src/server/migrations/20200417135519_crop_stages_update_foreign_fk_crop_id.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    table
+      .foreign('fk_crop_id')
+      .references('id')
+      .inTable('crops');
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.table('crop_stages', function(table) {
+    table.renameColumn('fk_crop_id', 'fk_crops_id');
```
Comment (moonflare) [inline src/server/migrations/20200417135519_crop_stages_update_foreign_fk_crop_id.js:14]:
```suggestion
    table.dropForeign('fk_crop_id');
    table.renameColumn('fk_crop_id', 'fk_crops_id');
```

**Comment by moonflare** [comment]
Ah, I was too late. We will need another PR to fix the down migration script. @dpfernandes @cecastosic @mahieakhtar

**Comment by mahieakhtar** [comment]
> Ah, I was too late. We will need another PR to fix the down migration script. @dpfernandes @cecastosic @mahieakhtar

Thanks for your review Dan.
Updated down migration and created new PR.Please review it again.Thank you

**Pull request merged**